lib/faker/date.ex

```diff
@@ -47,15 +45,26 @@ if Version.match?(System.version(), ">= 1.3.0") do
     """
     @spec forward(integer) :: Date.t
     def forward(days) do
-      unix_now =
-        DateTime.utc_now()
-        |> DateTime.to_unix()
+      Faker.DateTime.forward(days)
+      |> DateTime.to_date
+    end
+
+    @doc """
+    Returns a random date between two dates, start date and end date not
+    included
+    """
+    @spec between(Date.t, Date.t) :: Date.t
+    def between(from, to) do
+      Faker.DateTime.between(to_datetime(from), to_datetime(to))
+      |> DateTime.to_date
+    end
 
-      sign = if days < 0, do: -1, else: 1
+    # private
```

Comment:
Minor nitpick but I don't like comments unless they're really required: `defp` (used to define the method below) already indicates a method is private and therefore this comment doesn't add anything. It also isn't a convention to use a `# private`. Additionally, using a `# private` comments implies that all private methods are required to be defined below that line, while I prefer private methods are defined close to the methods that use them. (which isn't that uncommon and appears also to be the convention in the rest of the project)